src/win/atomicops-inl.h

```diff
@@ -34,7 +34,7 @@
 /* target to be aligned. */
 #pragma intrinsic(_InterlockedOr8)
 
-static char __declspec(inline) uv__atomic_exchange_set(char volatile* target) {
+static char INLINE uv__atomic_exchange_set(char volatile* target) {
```

Comment:
You should `#include "internal.h"` if you use INLINE.  It works by accident now because src/win/async.c includes it before it includes this file.

I don't know if there is a material difference between `__declspec(inline)` and `__inline` or `inline`.  Someone with more Windows experience should do the final sign-off.